music/demos/callback_player.html

```diff
@@ -0,0 +1,83 @@
+<html>
```

Comment:
you need to add callback_player.ts to build_demos.sh and add a link to callback_player.html from index.html

Reply:
done